packages/fxa-auth-server/lib/routes/subscriptions/stripe.ts

```diff
@@ -137,7 +137,6 @@ export class StripeHandler {
     await this.push.notifyProfileUpdated(uid, devices);
     this.log.notifyAttachedServices('profileDataChange', request, {
       uid,
-      email,
```

Comment:
It seems like email shouldn't have been sent here. But we should double check with RPs about this change.